modelexpress_client/python/modelexpress/load_strategy/trainer_pull_strategy.py

```
@@ -0,0 +1,234 @@
+                ),
+            )
+
+    def iter_param_shards(self, model: Any):
+        if hasattr(model, "named_parameters"):
+            yield from model.named_parameters()
+
+    def post_pull_hook(self, model: Any) -> None:
+        if self._ctx.adapter is not None and hasattr(self._ctx.adapter, "process_weights_after_loading"):
+            try:
+                self._ctx.adapter.process_weights_after_loading(model)
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)